### PR TITLE
Time range picker

### PR DESCRIPTION
**Pull request by Baribal02**
This is a component that allows a user to pick a time duration
It is for example used for reports form to get the total pause time.

image

### CODE REVIEW

File: app/src/main/java/ch/epfl/skysync/components/RangeTimePicker.kt [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    title: String,
+    showDialog: Boolean,
+    onDismiss: () -> Unit,
+    onConfirm: (Long) -> Unit
+) {
+  if (!showDialog) {
+    return
+  }
+  var timeDuration by remember { mutableLongStateOf(0L) }
+  val listState1 = rememberLazyListState(Int.MAX_VALUE / 2)
```
Comment (mathieumognol) [inline app/src/main/java/ch/epfl/skysync/components/RangeTimePicker.kt:43]: Could use more descriptive cariable name like for example: `hourListState`

File: app/src/main/java/ch/epfl/skysync/components/RangeTimePicker.kt [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  val listState2 = rememberLazyListState(Int.MAX_VALUE / 2)
```
Comment (mathieumognol) [inline app/src/main/java/ch/epfl/skysync/components/RangeTimePicker.kt:44]: Could use more descriptive cariable name like for example: `minuteListState`

**Review by mathieumognol — APPROVED** [review]
LGTM

File: app/src/main/java/ch/epfl/skysync/components/RangeTimePicker.kt (hunk 1 above)
Comment (Baribal02) [reply app/src/main/java/ch/epfl/skysync/components/RangeTimePicker.kt]: Done

**Pull request merged**